swupd/files.go

```diff
@@ -81,17 +81,38 @@ const (
 	AVX2_3
 	AVX512_2
 	AVX512_3
+	SSE_4
+	SSE_5
+	SSE_6
+	SSE_7
+	AVX2_5
+	AVX2_7
+	AVX512_6
+	AVX512_7
+	APX_4
+	APX_5
+	APX_6
+	APX_7
 )
 
 var modifierPrefixes = map[ModifierFlag]string{
 	SSE_0:    "",
 	SSE_1:    "",
 	SSE_2:    "",
 	SSE_3:    "",
+	SSE_4:    "",
 	AVX2_1:   "/V3",
 	AVX2_3:   "/V3",
+	AVX2_5:   "/V3",
+	AVX2_7:   "/V3",
 	AVX512_2: "/V4",
 	AVX512_3: "/V4",
+	AVX512_6: "/V4",
+	AVX512_7: "/V4",
+	APX_4:    "/V5",
```

Comment:
should this not be V5 but something like VA in case an actual v5 comes along that's something else